src/Web/Masa.Auth.Web.Admin.Rcl/Pages/Subjects/Users/UpdateUserDialog.razor

```diff
@@ -1,119 +1,135 @@
 ﻿@inherits AdminCompontentBase
 
-<MDialog Value="Visible" ValueChanged=UpdateVisible Width="800">
-    <MCard Class="pa-6">
-        <MForm Model=User EnableValidation>
-            <div class="d-flex">
-                <div class="mt-10 ml-auto mr-auto text-center" style="width:212px;">
-                    <MAvatar Size=80>
-                        <img src="/_content/Masa.Auth.Web.Admin.Rcl/img/subject/user.svg" />
-                    </MAvatar>
-                    <MTextField @bind-Value="User.DisplayName" Class="mt-0" HideDetails="@("auto")" Placeholder="@T("Give him a nickname!")" />
+<SheetDialog Value="Visible" ValueChanged=UpdateVisible Title="@T("Update User")">
+    <MForm Model=User EnableValidation @ref="Form">
+        <div style="padding-left:204px;padding-right:204px;padding-bottom:24px;padding-top:24px;height:100%;width: 100%;position: absolute;">
+             <div style="position: relative;" class="full-height">    
+                <div class="d-flex">
+                    <div class="mx-auto text-center" style="width:212px;">
+                        <MAvatar Size=80>
+                            <img src="/_content/Masa.Auth.Web.Admin.Rcl/img/subject/user.svg" />
```

Comment:
cdn

Reply:
This is written temporarily, the latter is not a dead url